ColorCodeProj.c

```diff
@@ -0,0 +1,65 @@
+/*********************************************/
+/***File   : ColorCodeProj.c
+/***Author : Naveen B 
+/***brief  : Color code project project file
+/*********************************************/
+/*------ standard includes -------*/
+#include <stdio.h>
+#include <assert.h>
+
+/*------ project spec includes --------*/
+#include "ColorCodeProj.h"
+
+void ColorPairToString(const ColorPair* colorPair, char* buffer) {
+    sprintf(buffer, "%s %s",
+        MajorColorNames[colorPair->majorColor],
+        MinorColorNames[colorPair->minorColor]);
+}
+
+ColorPair GetColorFromPairNumber(int pairNumber) {
+    ColorPair colorPair;
+    int zeroBasedPairNumber = pairNumber - 1;
+    colorPair.majorColor = 
+        (enum MajorColor)(zeroBasedPairNumber / (int)NUM_OF_MINOR_COLOR);
+    colorPair.minorColor =
+        (enum MinorColor)(zeroBasedPairNumber % (int)NUM_OF_MINOR_COLOR);
+    return colorPair;
+}
+
+int GetPairNumberFromColor(const ColorPair* colorPair) {
+    return colorPair->majorColor * (int)NUM_OF_MINOR_COLOR +
+            colorPair->minorColor + 1;
+}
+
+void testNumberToPair(int pairNumber,
```

Comment:
Think of test-names that include their expectation. Do you see any possibilities for more tests?